<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorBarWidgetClass</class>
 <widget class="QWidget" name="ColorBarWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>108</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorBarWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbScaleType"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblN">
         <property name="text">
          <string>n =</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dspnN"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MantidQt::API::QScienceSpinBox" name="valMax">
       <property name="toolTip">
        <string>Maximum value in the color scale</string>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MantidQt::API::QScienceSpinBox" name="valMin">
       <property name="toolTip">
        <string>Minimum value in the color scale</string>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoScaleForCurrentSlice">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When this is checked the color scale range will be automatically set to the current slice whenever the user zooms to a new peak, pans through the view or changes the slice with the slider control&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Autoscale to
current slice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoScale">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This flag signals that the color scale range should be set automatically to the current slice range when a workspace is loaded. Note that auto scaling will be applied when a workspace is loaded for the very first time. This option is mainly relevant for live data workspaces, which are continuously being updated and reloaded.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Autoscale
on load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MantidQt::API::QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>MantidQtWidgets/Common/QScienceSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
